File: Swift/프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.swift
import Foundation

func gcd(_ a: Int, _ b: Int) -> Int {
    if b == 0 {
        return a
    } else {
        return gcd(b, a % b)
    }
    
}

func lcm(_ a: Int, _ b: Int) -> Int {
    return a * b / gcd(a, b)
}

func solution(_ n:Int) -> Int {
    return lcm(n, 6) / 6
}

File: Python3/프로그래머스/0/120815. 피자 나눠 먹기 （2）/피자 나눠 먹기 （2）.py
def gcd(a, b):
    while b > 0:
        a, b = b, a % b
    return a

def lcm(a, b):
    return a * b // gcd(a, b)
    


def solution(n):
    answer = lcm(n, 6)//6
    
    return answer